docs/rules/0121/no-cycles.md

```diff
@@ -0,0 +1,129 @@
+---
+rule:
+  aip: 121
+  name: [core, '0121', no-cycles]
+  summary: Resources must not form a resource reference cycle.
+permalink: /121/no-cycles
+redirect_from:
+  - /0121/no-cycles
+---
+
+# Resource must support get
+
+This rule enforces that resources do not create reference cycles as mandated in
```

Comment:
```suggestion
This rule enforces that resources do not create cycles of mutable references as mandated in
```

Reply:
slightly clearer wording, wdyt?

Reply:
Yeah for sure. Done in a separate commit.